iGOT/client-assets/assets/configurations/profile-nudge.json

```diff
@@ -77,18 +89,24 @@
           ]
         },
         {
-          "startTime": 16,
-          "endTime": 19,
+          "startTime": 15,
+          "endTime": 17,
           "greet": "Namaste, <userName>!",
           "webGreet": "slot1Greet",
           "textColor": "#000000",
-          "webInfo": ["slot4Info1", "slot4Info2", "slot4Info3", "slot4Info4", "slot4Info5"],
+          "webInfo": ["slot4Info1", "slot4Info2", "slot4Info3", "slot4Info4", "slot4Info5", "slot4Info6", "slot4Info7", "slot4Info8", "slot4Info9", "slot4Info10", "slot4Info11"],
           "info": [
-            "Invest in yourself. Do a mandatory course on the iGOT mobile app.",
-            "Finished work? Join the iGOT social media community - on LinkedIn, X, YouTube, Instagram, Facebook and Spotify.",
-            "Congratulations! You belong to the world's largest community of government e-Learners.",
-            "Chai o'clock! Stay ahead in your career journey.",
-            "Lead by example! Show your colleagues the power of learning with iGOT."
+            "Lunch done, energy up! Dive into the AI course and be tech-educated!",
+            "Chai o'clock! Sip up, watch the latest ‘Karmayogi Talks’",
+            "Get, set, and go for non-stop learning with 1100+ courses!",
+            "Top government case studies await at ‘Gyaan Karmayogi’!",
+            "Liked a course, recommend it to your peers at iGOT!",
```

Comment:
Please confirm text for hindi as well, also are we having different numbers number of messages for each slot like 7, 13

Reply:
updated the PR with hindi text, which is provided by Palash